flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/JobVertexFlameGraphInfo.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.rest.messages;
+
+import org.apache.flink.runtime.rest.handler.job.JobVertexFlameGraphHandler;
+import org.apache.flink.runtime.webmonitor.threadinfo.OperatorFlameGraph;
+
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonCreator;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonInclude;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonProperty;
+
+/** Response type of the {@link JobVertexFlameGraphHandler}. */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class JobVertexFlameGraphInfo implements ResponseBody {
+
+    public static JobVertexFlameGraphInfo empty() {
+        return new JobVertexFlameGraphInfo(-1, null);
+    }
+
+    private static final String FIELD_NAME_END_TIMESTAMP = "end-timestamp";
```

Comment:
```suggestion
    private static final String FIELD_NAME_END_TIMESTAMP = "endTimestamp";
```
We are trying to phase out dashes

Reply:
This field it only supposed to be used in JSON and such approach seems to be an established practice:
https://github.com/apache/flink/blob/dcb9f83d4ad3710bb9e34e3de3bb82a146ff2050/flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/JobVertexBackPressureInfo.java#L42

Reply:
As I said, camelCase is what we use for new fields. Dashes are basically just a relic from very old handlers (which also handled things inconsistently).
To reduce the amount of boilerplate we'll need once we migrate all fields please do not use dashes.

Reply:
Changed as requested.